README.md

```diff
@@ -29,5 +29,11 @@ Import-Module -Name '.\Carbon.Windows'
 ## Commands
 
 * `Get-CBackConnectionHostName`, for getting the list of currently configured back connection hostnames.
+* `Get-CEventLog`, as a drop in replacement for using `Get-EventLog` in PowerShell 7.
+* `Install-CEventLog`, which creates an event log only if it doesn't already exist.
+* `New-CEventLog`, as a drop in replacement for using `New-EventLog` in PowerShell 7.
 * `Register-CBackConnectionHostName`, for adding a back connection hostname, if it isn't already configured.
+* `Remove-CEventLog`, as a drop in replacement for using `Remove-EventLog` in PowerShell 7.
+* `Test-CEventLog`, which checks if an event log exists.
+* `Uninstall-CEventLog`, which removes an event log only if it doesn't already exist

```

Comment:
Remove the comments about being drop-in replacements for other functions and just use a short description of each.